python/ui/setup_project.py

```diff
@@ -99,66 +99,6 @@ def setupUi(self, Wizard):
         self.verticalLayout.addItem(spacerItem4)
         self.verticalLayout.setStretch(13, 1)
         Wizard.addPage(self.setup_type_page)
-        self.default_configs_page = DefaultConfigPage()
```

Comment:
This change should be accompanied with an update to the resources folder. [setup_project.py is auto-generated](https://github.com/shotgunsoftware/tk-framework-adminui/blob/master/resources/build_resources.sh#L42).